src/Microsoft.ML.FastTree/FastTreeClassification.cs

```diff
@@ -39,7 +40,7 @@
     "FastTree Binary Executor",
     FastTreeBinaryPredictor.LoaderSignature)]
 
-namespace Microsoft.ML.Runtime.FastTree
+namespace Microsoft.ML.Trainers
```

Comment:
>Trainers [](start = 23, length = 8)

I could be wrong, but my (possibly flawed) understanding was that we would have the *extension*methods for the `MLContext` objects in `Microsoft.ML.Trainers`, while keeping the actual implementing classes properly encapsulated still in the subnamespaces. I would prefer that the classes for FastTree *not* live in bifurcated namespaces.